web/src/components/pages/error-page/error-page.css

```diff
@@ -42,13 +43,18 @@
         margin: 0 auto;
         padding: 80px 40px 100px 20px;
 
-        @media (--sm-down) {
-            padding-bottom: 20px;
+        @media (--xs-down) {
+            padding-top: 32px;
+            padding-bottom: 40px;
         }
     }
 
-    .error-page-container-text {
+    .error-page-container-inner > * {
```

Comment:
Hmm, looking at the template code for that, can this just be a `> div`? While the performance hit of `*` selectors is probably negligible on most modern systems, it's always good to reduce their use where possible.

Reply:
No problem, I just replaced this with the specific child classes. For what it's worth, `> *` was an intentional decision – I think it reads clearer, and I haven't seen compelling evidence that selectors are a bottleneck. But either is fine by me!

Reply:
Totally fair! Next 1:1 let's start hammering out some CSS guidelines maybe, since we want to start focusing on frontend / accessibility in the next little while.